gaas-ui/src/rest/repositories/create-federated-graph-repo.ts

```diff
@@ -8,7 +8,7 @@ export class CreateFederatedGraphRepo {
         graphId: string,
         description: string,
         configName: string,
-        config: ICreateGraphConfig,
+        config: ICreateGraphConfig
```

Comment:
Removed the comma here but on line 20 a comma is added, is there a difference?

Reply:
I think it might be because one is a function and the other is an object